models/linkages/R/write_restart.LINKAGES.R

```diff
@@ -343,8 +345,8 @@ write_restart.LINKAGES <- function(outdir, runid, start.time, stop.time, setting
   
   # make a new settings with the right years min start date and end date - fail in informative way
   
-  settings$run$start.date <- paste0(start.time + 1, "/01/01")
-  settings$run$end.date <- paste0(stop.time + 1, "/12/31")
+  settings$run$start.date <- paste0(formatC(year(start.time + 1), width = 4, format = "d", flag = "0"), "/01/01")
+  settings$run$end.date <- paste0(formatC(year(stop.time), width = 4, format = "d", flag = "0"), "/12/31")
```

Comment:
was this supposed to be `stop.time+1` as well?

Reply:
No because it's +1 in sda.enkf main function. But, I'll have to make sure all the time logic makes sense when I'm refactoring. Thanks for noticing!